### DISCUSSION TITLE
Confused about `input-sum-value` and `output-sum-value`

### DISCUSSION THREAD

**Discussion by oceanumeric** [cosmograph-org/cosmograph-issues#8] [General]
In the docs - node appearance part [here](https://cosmograph.app/docs/cosmograph/Sharing%20Graphs#node-appearance) 
I am not sure what does `input` and `output` mean? 

`input-sum-value`: Colors/sizes nodes by the sum across input-only records in the value column
`output-sum-value`: Colors/sizes nodes by the sum across output-only records in the value column

input means source and output means target?

Why set sum? Since I am not sure the meaning, also could not get the case when do we need this.

Thanks :)

**Comment by kolmakova** [comment]
Hello, @oceanumeric! The `input-sum-value` option calculates the sum of values by `value` column from all the input links of the node (e.g. link records where this node is a target). Similarly, the `output-sum-value` option calculates the sum of values from all the output links (link records where the node is a source).

By visually representing the cumulative value associated with the nodes, you can easily identify nodes with higher or lower cumulative values based on their input or output connections. 

This feature can be particularly useful in exploring transactions or networks where you want to identify nodes that have a high input value but do not have a corresponding output value or vice versa. For example, in the context of wallets, you can use this to find wallets that receive a significant amount of funds but have few outgoing transactions, which may indicate an anomaly within that network.

Hope this answer helps you figure it out. ☺️
